package.json

```diff
@@ -97,6 +97,7 @@
     "test:server": "make .venv && cd test/server && ../../.venv/bin/pytest -m 'not slow'",
     "make_versions_exact": "node bin/build-scripts/make_versions_exact.js",
     "update_outdated": "david update && npm run make_versions_exact && npm update && npm outdated",
-    "fill_database": "make .venv && ./.venv/bin/python  ./bin/load_test_exercise.py -q --little-image --create=50 --read-shot=0 --read-my-shots=0 --search=0 --new-account http://localhost:10080"
+    "fill_database": "make .venv && ./.venv/bin/python  ./bin/load_test_exercise.py -q --little-image --create=50 --read-shot=0 --read-my-shots=0 --search=0 --new-account http://localhost:10080",
+    "svgo": "for filename in $(find addon static -name '*.svg') ; do svgo -i $filename ; done"
```

Comment:
Why not just something like this?
```js
"svgo --folder=addon/webextension/icons -q && svgo --folder=static/img -q"
```
(I can't find a way to pass multiple input folders yet without it blowing up or assuming the second folder is a dist directory.)

---

```sh
time (svgo --folder=addon/webextension/icons -q && svgo --folder=static/img -q)
# 2.16s user 0.28s system 110% cpu 2.196 total

# vs 

time (for filename in $(find addon static -name '*.svg') ; do svgo -i $filename -q ; done)
# 45.85s user 7.21s system 99% cpu 53.244 total
```